demovieworderhooks/demovieworderhooks.php

```diff
@@ -0,0 +1,222 @@
+{
+    public function __construct()
+    {
+        $this->name = 'demovieworderhooks';
+        $this->author = 'PrestaShop';
+        $this->version = '1.0.0';
+        $this->displayName = 'Demonstration of new hooks in PrestaShop 1.7.7 order view page';
+        $this->ps_versions_compliancy = ['min' => '1.7.7.0', 'max' => _PS_VERSION_];
+
+        parent::__construct();
+    }
+
+    public function install()
+    {
+        if (!parent::install()) {
+            return false;
+        }
+
+        $installer = InstallerFactory::create();
+
+        return $installer->install($this);
+    }
+
+    public function uninstall()
+    {
+        $installer = InstallerFactory::create();
+
+        return $installer->uninstall() && parent::uninstall();
+    }
+
+    /**
+     * Displays customer's signature.
+     */
+    public function hookDisplayBackOfficeOrderActions(array $params)
+    {
+        /** @var SignatureRepository $signatureRepository */
+        $signatureRepository = $this->get('prestashop.module.demovieworderhooks.repository.signature_repository');
+
+        /** @var SignaturePresenter $signaturePresenter */
+        $signaturePresenter = $this->get('prestashop.module.demovieworderhooks.presenter.signature_presenter');
+
+        $signature = $signatureRepository->findOneBy(['orderId' => $params['id_order']]);
+
+        if (!$signature) {
+            return '';
+        }
+
+        return $this->render($this->getModuleTemplatePath() . 'customer_signature.html.twig', [
+            'signature' => $signaturePresenter->present($signature, (int) $this->context->language->id),
+        ]);
+    }
+
+    public function hookDisplayAdminOrderContentOrder(array $params)
+    {
+        return 'displayAdminOrderContentOrder';
+    }
+
+    /**
+     * Display shipment tracking information.
+     */
+    public function hookDisplayAdminOrderTabContent(array $params)
+    {
+        /** @var PackageLocationRepository $locationRepository */
+        $locationRepository = $this->get('prestashop.module.demovieworderhooks.repository.package_location_repository');
+
+        $locations = $locationRepository->findBy(
+            ['orderId' => $params['id_order']],
+            ['position' => 'asc']
+        );
+
+        /** @var PackageLocationsPresenter $locationsPresenter */
+        $locationsPresenter = $this->get('prestashop.module.demovieworderhooks.presenter.package_locations_presenter');
+
+        return $this->render($this->getModuleTemplatePath() . 'tracking.html.twig', [
+            'packageLocations' => $locationsPresenter->present($locations),
+        ]);
+    }
+
+    /**
+     * Display tracking tab link.
+     */
+    public function hookDisplayAdminOrderTabLink(array $params)
+    {
+        return $this->render($this->getModuleTemplatePath() . 'tracking_link.html.twig');
+    }
+
+    public function hookDisplayAdminOrderMain(array $params)
+    {
+        // ERP integration
+        return 'displayAdminOrderMain';
```

Comment:
Same here, lets put a small image (a prestashop puffin) with a text "Here is a nice location to display integration status for an ERP for example"

Reply:
ah, I already have the code for this, my mistake not committing/pushing it. Will update soon!